_tests/integration/mac_test.go

```diff
@@ -113,9 +109,6 @@ configs:
               run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
           - git-clone@%s: {}
           - cache-pull@%s: {}
-          - script@%s:
-              title: Do anything with Script step
-          - certificate-and-profile-installer@%s: {}
```

Comment:
Mac Steps still need certificate and profile installer as there is no other Option available.

Reply:
Thanks for catching this.